node.gypi

```diff
@@ -312,6 +301,9 @@
       'ldflags': [ '-Wl,-z,relro',
                    '-Wl,-z,now' ]
     }],
+    [ 'OS=="linux" and target_arch=="x64" and node_use_large_pages=="true"', {
+      'ldflags': [ '-Wl,-T <(PRODUCT_DIR)/../../src/large_pages/ld.implicit.script' ]
```

Comment:
Simple `'-Wl,-T src/large_pages/ld.implicit.script'` doesn't work?
Or `'-Wl,-T', 'src/large_pages/ld.implicit.script'` doesn't work?

Reply:
Yes we tried `-Wl,-T src/large_pages/ld.implicit.script` and that didnt work. Looks like `-T` needs full path or some `-L` with directories to search for.

Reply:
Thank you.
It's not going to be a problem with `make` and linux, but FTR it will not work with `ninja`, `msvs` or cross compilers.